enterprise/coderd/provisionerdaemons.go

```diff
@@ -74,6 +82,24 @@ func (api *API) provisionerDaemons(rw http.ResponseWriter, r *http.Request) {
 	httpapi.Write(ctx, rw, http.StatusOK, db2sdk.List(daemons, db2sdk.ProvisionerDaemon))
 }
 
+func provisionerTags(r *http.Request) map[string]string {
+	tags := r.URL.Query()["tags"]
+	if len(tags) == 0 {
+		return nil
+	}
+
+	tagMap := map[string]string{}
+	for _, tag := range tags {
+		parts := strings.SplitN(tag, "=", 2)
+		if len(parts) == 2 {
+			// TODO(sas): seems like the kind of place where nilpointers would pop up.
+			tagMap[parts[0]] = parts[1]
+		}
```

Comment:
Else error?

Reply:
This entire function is now defunct, thanks to some updates proposed by Cian.